governance/votes/Executive vote - May 14, 2021.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Template - [Executive Vote] Parameter Changes and Oracle Updates - May 14, 2021
+
+---
+# [Executive Proposal] Parameter Changes and Oracle Updates - May 14, 2021
+
+The Governance Facilitator(s) and the MakerDAO Smart Contracts Domain Team have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

Comment:
Smart Contracts -> Protocol Engineering? We should update the template for that too.

Reply:
Ope yup on it